src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasureRecord.ts

```diff
@@ -0,0 +1,34 @@
+import { ContentRecord } from "@activeviam/activeui-sdk-5.0";
+import { produce } from "immer";
+
+/**
+ * Transforms the serialized definition of a calculated measure created with ActiveUI 5.0, into one that is natively supported by ActivePivot.
+ */
+export const migrateCalculatedMeasureRecord = (
+  legacyCalculatedMeasureContent: ContentRecord,
+  calculatedMeasureName: string,
+): ContentRecord => {
+  const migratedRecord = produce(legacyCalculatedMeasureContent, (draft) => {
+    const parsedContent = JSON.parse(draft.entry.content);
+
+    const expression = parsedContent.expression;
+    const properties: string[] = parsedContent.properties;
+
+    const formatStringExpression = properties
+      ? properties.find((property) => property.startsWith("FORMAT_STRING"))
```

Comment:
In response to [this comment](https://github.com/activeviam/activeui-migration/pull/45#discussion_r1007975828) on the earlier version of this function
> Is there a reason why it's `startsWith` and not a strict equality check `===` ?

`properties` is an array of strings, for example `FORMAT_STRING = "#,###.##"`